<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProWindow</class>
 <widget class="QMainWindow" name="ProWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>452</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>452</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>452</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>专业模式</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../../../Pictures/icon.png</normaloff>../../../../Pictures/icon.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="process_button">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>260</y>
      <width>100</width>
      <height>60</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>100</width>
      <height>60</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>100</width>
      <height>60</height>
     </size>
    </property>
    <property name="text">
     <string>数据处理</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>200</y>
      <width>341</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>80</y>
      <width>102</width>
      <height>88</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="load_file_button">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>40</height>
        </size>
       </property>
       <property name="text">
        <string>加载文件</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="load_dir_button">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>40</height>
        </size>
       </property>
       <property name="text">
        <string>加载文件夹</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="Line" name="line_4">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>350</y>
      <width>341</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>400</y>
      <width>208</width>
      <height>63</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="1" column="1" alignment="Qt::AlignHCenter">
      <widget class="QSpinBox" name="out_amount">
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>70</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QRadioButton" name="multiple_sv_button">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>多文件保存</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">save_mode_BG</string>
       </attribute>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;输出光谱数量&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QRadioButton" name="single_sv_button">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>单文件保存</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">save_mode_BG</string>
       </attribute>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="load_label">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>40</y>
      <width>341</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;等待加载&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="save_file_button">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>400</y>
      <width>100</width>
      <height>60</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>100</width>
      <height>60</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>100</width>
      <height>60</height>
     </size>
    </property>
    <property name="text">
     <string>保存</string>
    </property>
   </widget>
   <widget class="Line" name="line_5">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>30</y>
      <width>341</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>70</y>
      <width>210</width>
      <height>122</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QRadioButton" name="renishaw_select_button">
       <property name="text">
        <string>Renishaw</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">load_type_BG</string>
       </attribute>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QRadioButton" name="ezraman_select_button">
       <property name="text">
        <string>Ez Raman</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">load_type_BG</string>
       </attribute>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QRadioButton" name="iramanplus_select_button">
       <property name="text">
        <string>i Raman Plus</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">load_type_BG</string>
       </attribute>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QRadioButton" name="other_select_button">
       <property name="text">
        <string>自定义</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">load_type_BG</string>
       </attribute>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLineEdit" name="header_line">
       <property name="text">
        <string>表头</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="sep_line">
       <property name="text">
        <string>分隔符</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLineEdit" name="shift_line">
       <property name="text">
        <string>位移坐标</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="spectrum_line">
       <property name="text">
        <string>光谱坐标</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>250</y>
      <width>181</width>
      <height>101</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="0">
      <widget class="QRadioButton" name="method1_button">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>方法1</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">classify_mode_BG</string>
       </attribute>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QRadioButton" name="method2_button">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>方法2</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">classify_mode_BG</string>
       </attribute>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QRadioButton" name="method3_button">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>方法3</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">classify_mode_BG</string>
       </attribute>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QRadioButton" name="no_classify_button">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>不进行筛选</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">classify_mode_BG</string>
       </attribute>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QRadioButton" name="bs_remove_button">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>去基线</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1" alignment="Qt::AlignHCenter">
      <widget class="QLineEdit" name="peak_location">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>70</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>筛选峰位</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="process_label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>220</y>
      <width>341</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;等待处理&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="save_label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>370</y>
      <width>341</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;等待保存&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="save_mode_BG"/>
  <buttongroup name="load_type_BG"/>
  <buttongroup name="classify_mode_BG"/>
 </buttongroups>
</ui>
